ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -441,16 +443,20 @@ void completeRead() {
      * Update chunking and size related metrics - blob size, chunk count, and whether the blob is simple or composite.
      */
     private void updateChunkingAndSizeMetricsOnSuccessfulGet() {
-      routerMetrics.getBlobSizeBytes.update(bytesWritten);
+      routerMetrics.getBlobSizeBytes.update(bytesWritten.get());
       routerMetrics.getBlobChunkCount.update(numChunksTotal);
       if (options != null && options.getBlobOptions.getRange() != null) {
-        routerMetrics.getBlobWithRangeSizeBytes.update(bytesWritten);
+        routerMetrics.getBlobWithRangeSizeBytes.update(bytesWritten.get());
         routerMetrics.getBlobWithRangeTotalBlobSizeBytes.update(totalSize);
       }
-      if (numChunksTotal == 1) {
-        routerMetrics.simpleBlobGetCount.inc();
+      if (!options.getBlobOptions.isRawMode()) {
+        if (numChunksTotal == 1) {
+          routerMetrics.simpleBlobGetCount.inc();
+        } else {
+          routerMetrics.compositeBlobGetCount.inc();
+        }
       } else {
-        routerMetrics.compositeBlobGetCount.inc();
+        // TODO: how to track these counts in raw mode?
```

Comment:
i think you can just another metric `rawBlobGetCount` (don't know if it is worth trying to determine simple v/s composite v/s data)

Reply:
Adding that.  Think a single one is okay.